framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -256,62 +257,69 @@ int cfg_get_management_mode();
  *
  * @return BPL_CERTIFICATION_MODE_ON Certification mode is ON.
  * @return BPL_CERTIFICATION_MODE_OFF Certification mode is OFF.
+ * @return default_value if value not set
  * @return -1 Error.
  */
-int cfg_get_certification_mode();
+int cfg_get_certification_mode(int default_value = -1);
```

Comment:
Again,  no need to change the function prototype. The default value should be false.